lib/meow_nx/utils.ex

```diff
@@ -410,4 +410,24 @@ defmodule MeowNx.Utils do
 
     n
   end
+
+  @doc """
+  """
+  defn random_seed do
+    0
+  end
+
+  @doc """
+  Produce a random tensor with the given shape.
+
+  Mirrors the deprecated `MeowNx.Utils.random_uniform` so it's easier to upgrade.
+  """
+  defn random_uniform(shape, min \\ 0, max \\ 1) do
+    {result, _seed} =
+      random_seed()
```

Comment:
This is fine as a placeholder as you upgrade. Later I can look into proper support, which would involve passing the seed at the highest level and then propagating throughout the whole pipeline.

```suggestion
      0
```